### PR TITLE
py-netcdf4: adding version 1.6.2

### PR DESCRIPTION
**Pull request by snehring**
Simple version update.

### CODE REVIEW

File: var/spack/repos/builtin/packages/py-netcdf4/package.py [hunk 1]
```diff
@@ -14,6 +14,7 @@ class PyNetcdf4(PythonPackage):
 
     maintainers("skosukhin")
 
+    version("1.6.2", sha256="0382b02ff6a288419f6ffec85dec40f451f41b8755547154c575ddd9f0f4ae53")
     version("1.5.8", sha256="ca3d468f4812c0999df86e3f428851fb0c17ac34ce0827115c246b0b690e4e84")
     version("1.5.3", sha256="2a3ca855848f4bbf07fac366da77a681fcead18c0a8813d91d46302f562dc3be")
     version("1.4.2", sha256="b934af350459cf9041bcdf5472e2aa56ed7321c018d918e9f325ec9a1f9d1a30")
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/py-netcdf4/package.py:20]: setuptools should now be 41.2+

**Review by adamjstewart — CHANGES_REQUESTED** [review]

**Review by adamjstewart — APPROVED** [review]
LGTM, will give @skosukhin a chance to review. Ping me if you haven't heard anything back in a week and I'll merge.

**Review by skosukhin — APPROVED** [review]

File: var/spack/repos/builtin/packages/py-netcdf4/package.py [hunk 2]
```diff
@@ -14,6 +14,7 @@ class PyNetcdf4(PythonPackage):
 
     maintainers("skosukhin")
 
+    version("1.6.2", sha256="0382b02ff6a288419f6ffec85dec40f451f41b8755547154c575ddd9f0f4ae53")
```
Comment (tldahlgren) [inline var/spack/repos/builtin/packages/py-netcdf4/package.py:17]: Confirmed the sha256.

**Comment by snehring** [comment]
@spackbot re-run pipeline

**Pull request merged**